ai-service/src/ai_service/errors.py

```
@@ -0,0 +1,44 @@
+class AIServiceError(Exception):
+    """Base exception for all AI service errors."""
+
+    pass
```

Ruff (0.12.2)

Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)